src/buttons/css/buttons.css

```diff
@@ -38,7 +38,8 @@
 }
 .pure-button-active,
 .pure-button:active {
-    box-shadow: 0 0 0 1px rgba(0,0,0, 0.15) inset, 0 0 6px rgba(0,0,0, 0.20) inset;
+    box-shadow: 0 0 10px 1px rgba(0,0,0,.15) inset,0 0 6px rgba(0,0,0,.2) inset;
+    *font-weight: bold; /* since box-shadow wont show in IE < 8, make the font bold instead to denote active. */
```

Comment:
Seems like a good PR. We can get around the bolding issue by not worrying about IE7, for which we are dropping support .